docs/ja/core-concepts/ngsiv2-vs-ngsild.md

```
@@ -783,55 +806,71 @@ curl http://localhost:3000/ngsi-ld/v1/entities/urn:ngsi-ld:Sensor:S888 \
-2. 登録情報から `http://legacy-system.example.com/v2` を特定します
-3. NGSIv2 プロトコルを使用してクエリを転送します: `GET /v2/entities/urn:ngsi-ld:Sensor:S888`4. レスポンスを NGSIv2 → 内部フォーマット → NGSI-LD に変換し、クライアントに返します
+2. 登録情報から `http://legacy-system.example.com/v2` を識別します
+3. NGSIv2 プロトコルを使用してクエリを転送します: `GET /v2/entities/urn:ngsi-ld:Sensor:S888`4. NGSIv2 からのレスポンスを内部フォーマット → NGSI-LD に変換してクライアントに返します
 
 ---
 
-## ユースケースとベストプラクティス
+# ユースケースとベストプラクティス
 
-### どの API を使用すべきか？
+### どの API を使うべきか?
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)